fixed-requirements.txt

```diff
@@ -12,7 +12,7 @@ six==1.9.0
 pyyaml>=3.11,<4.0
 requests[security]>=2.11.1,<2.12
 apscheduler>=3.0.5,<3.1
-gitpython==0.3.2.1
+gitpython>=0.3.7,<1.0
```

Comment:
I just saw we follow that thingy everywhere.

Will `gitpython>=1.0.0,<1.1` work better? It's a bit more strict. Something in between the breaking change and bug-fix patch.